core/scripts/PrecisionErrors.js

```diff
@@ -243,13 +255,13 @@ async function runExport() {
   assert.equal(parseFloat(actualFeeMultiplier.toString()) / 1e18, 1.0);
 
   // Test 4) Raw collateral and actual collateral amount are the same to start.
-  assert.equal(startingRawContractCollateral.toString(), startingContractCollateral.toString());
+  assert.equal(rawContractCollateral.toString(), contractCollateral.toString());
 
   // Log results.
-  breakdown.expected = new CollateralBreakdown(startingContractCollateral);
-  breakdown.credited = new CollateralBreakdown(startingAdjustedContractCollateral);
-  breakdown.raw = new CollateralBreakdown(startingRawContractCollateral);
-  breakdown.feeMultiplier = new CollateralBreakdown(actualFeeMultiplier);
+  breakdown["Collateral owned by Contract"] = new CollateralBreakdown(contractCollateral);
```

Comment:
Did a lot of renaming of the properties in the `breakdown` object for better console.table printing. I will attach a sample console print to the comments